IO/d4est_vtk.c

```diff
@@ -1113,7 +1113,7 @@ d4est_vtk_write_point_scalar (d4est_vtk_context_t * cont,
   if (output_type == D4EST_VTK_ASCII){
   for (il = 0; il < Npoints; ++il) {
     ddl = use_nodes ? ntc[il] : il;
-    P4EST_ASSERT (0 <= ddl && ddl < Ncorners);
+    // P4EST_ASSERT (0 <= ddl && ddl < Ncorners);
```

Comment:
I commented out a few of these calls, since `Ncorners` isn't actually set in this scope. Perhaps this is a remnant of debugging with `P4EST_ENABLE_DEBUG`?

Reply:
Yeah this code file needs to be cleaned up and maybe then the binary bug could be found easier. I basically took p4est_vtk.c and added collocation point functionality, made the interface better and called it d4est_vtk.c So there might be a lot of residual un-needed code. Anything with P4EST_ASSERT is not mine.